python/test/Ice/optional/AllTests.py

```diff
@@ -424,100 +424,98 @@ def allTests(helper, communicator):
     test(p2 is True and p3 is True)
     f = initial.opBoolAsync(True)
     (p2, p3) = f.result()
     test(p2 is True and p3 is True)
 
-    (p2, p3) = initial.opShort(Ice.Unset)
-    test(p2 is Ice.Unset and p3 is Ice.Unset)
+    (p2, p3) = initial.opShort(None)
+    test(p2 is None and p3 is None)
     (p2, p3) = initial.opShort(56)
     test(p2 == 56 and p3 == 56)
     f = initial.opShortAsync(56)
     (p2, p3) = f.result()
     test(p2 == 56 and p3 == 56)
 
-    (p2, p3) = initial.opInt(Ice.Unset)
-    test(p2 is Ice.Unset and p3 is Ice.Unset)
+    (p2, p3) = initial.opInt(None)
+    test(p2 is None and p3 is None)
     (p2, p3) = initial.opInt(56)
     test(p2 == 56 and p3 == 56)
     f = initial.opIntAsync(56)
     (p2, p3) = f.result()
     test(p2 == 56 and p3 == 56)
 
-    (p2, p3) = initial.opLong(Ice.Unset)
-    test(p2 is Ice.Unset and p3 is Ice.Unset)
+    (p2, p3) = initial.opLong(None)
+    test(p2 is None and p3 is None)
     (p2, p3) = initial.opLong(56)
     test(p2 == 56 and p3 == 56)
     f = initial.opLongAsync(56)
     (p2, p3) = f.result()
     test(p2 == 56 and p3 == 56)
 
-    (p2, p3) = initial.opFloat(Ice.Unset)
-    test(p2 is Ice.Unset and p3 is Ice.Unset)
+    (p2, p3) = initial.opFloat(None)
+    test(p2 is None and p3 is None)
     (p2, p3) = initial.opFloat(1.0)
     test(p2 == 1.0 and p3 == 1.0)
     f = initial.opFloatAsync(1.0)
     (p2, p3) = f.result()
     test(p2 == 1.0 and p3 == 1.0)
 
-    (p2, p3) = initial.opDouble(Ice.Unset)
-    test(p2 is Ice.Unset and p3 is Ice.Unset)
+    (p2, p3) = initial.opDouble(None)
+    test(p2 is None and p3 is None)
     (p2, p3) = initial.opDouble(1.0)
     test(p2 == 1.0 and p3 == 1.0)
     f = initial.opDoubleAsync(1.0)
     (p2, p3) = f.result()
     test(p2 == 1.0 and p3 == 1.0)
 
-    (p2, p3) = initial.opString(Ice.Unset)
-    test(p2 is Ice.Unset and p3 is Ice.Unset)
+    (p2, p3) = initial.opString(None)
+    test(p2 is None and p3 is None)
     (p2, p3) = initial.opString("test")
     test(p2 == "test" and p3 == "test")
     f = initial.opStringAsync("test")
     (p2, p3) = f.result()
     test(p2 == "test" and p3 == "test")
 
-    (p2, p3) = initial.opMyEnum(Ice.Unset)
-    test(p2 is Ice.Unset and p3 is Ice.Unset)
+    (p2, p3) = initial.opMyEnum(None)
+    test(p2 is None and p3 is None)
     (p2, p3) = initial.opMyEnum(Test.MyEnum.MyEnumMember)
     test(p2 == Test.MyEnum.MyEnumMember and p3 == Test.MyEnum.MyEnumMember)
     f = initial.opMyEnumAsync(Test.MyEnum.MyEnumMember)
     (p2, p3) = f.result()
     test(p2 == Test.MyEnum.MyEnumMember and p3 == Test.MyEnum.MyEnumMember)
 
-    (p2, p3) = initial.opSmallStruct(Ice.Unset)
-    test(p2 is Ice.Unset and p3 is Ice.Unset)
+    (p2, p3) = initial.opSmallStruct(None)
+    test(p2 is None and p3 is None)
     p1 = Test.SmallStruct(56)
     (p2, p3) = initial.opSmallStruct(p1)
     test(p2 == p1 and p3 == p1)
-    (p2, p3) = initial.opSmallStruct(None)  # Test null struct
-    test(p2.m == 0 and p3.m == 0)
```

Comment:
This behavior change, previously using None was equivalent to using a default constructed struct. Now it means the param is unset.